datasketch/lsh.py

```diff
@@ -173,7 +173,3 @@ def __init__(self, threshold=0.9, sample_size=128, weights=(0.5,0.5)):
         (false_positive_weight, false_negative_weight).
         '''
         super(WeightedMinHashLSH, self).__init__(threshold, sample_size, weights)
-
-    def _H(self, hs):
```

Comment:
Please note that this is not needed anymore - the parent class method is universal.